### ISSUE TITLE
Property 'createUser' does not exist on type 'typeof CometChat'.

### ISSUE THREAD

**Issue by sarayut-kung** [cometchat-pro/cordova-ionic-chat-sdk#5]
Using Ionic 3 Project and following this 
https://prodocs.cometchat.com/docs/cordova-ionic-quick-start

Install
`npm install @cometchat-pro/cordova-ionic-chat@2.0.6 --save`

Use

```
  CometChat.createUser(user, this.apiKey).then(
            (user: any) => {
              console.log("user created", user);
            }, (error: any) => {
              console.log("error", error);
            }
   )
```

Error

`Property 'createUser' does not exist on type 'typeof CometChat'.`

Thank you.

**Comment by sarayut-kung** [comment]
But this code work.

**Comment by mayur-bhandari** [comment]
Hello @secretmoo35 ,

This has been fixed in our latest beta release v2.0.7-beta1.
